pyproject.toml

```diff
@@ -96,3 +96,39 @@ log_cli_level = "INFO"
 log_cli_format = "[%(levelname)s] %(name)s: %(message)s"
 # Custom markers
 markers = ["timeit"]
+
+[[tool.mypy.overrides]]
+module = "confluent_kafka.*"
+ignore_missing_imports = true
+
+[[tool.mypy.overrides]]
+module = "quixstreams.core.*"
+ignore_errors = true
+
+[[tool.mypy.overrides]]
+module = "quixstreams.dataframe.*"
+ignore_errors = true
+
+[[tool.mypy.overrides]]
+module = "quixstreams.models.*"
+ignore_errors = true
+
+[[tool.mypy.overrides]]
+module = "quixstreams.platforms.*"
+ignore_errors = true
+
+[[tool.mypy.overrides]]
+module = "quixstreams.sinks.community.*"
+ignore_errors = true
+
+[[tool.mypy.overrides]]
+module = "quixstreams.sources.community.*"
+ignore_errors = true
+
+[[tool.mypy.overrides]]
+module = "quixstreams.state.*"
+ignore_errors = true
+
+[[tool.mypy.overrides]]
+module = "quixstreams.rowproducer.*"
```

Comment:
This should do

```suggestion
module = [
    "quixstreams.core.*",
    "quixstreams.dataframe.*",
    "quixstreams.models.*",
    "quixstreams.platforms.*",
    "quixstreams.sinks.community.*",
    "quixstreams.sources.community.*",
    "quixstreams.state.*",
    "quixstreams.rowproducer.*"
]
```

Reply:
So many ignored 🙈
Too much work to fix those?

Reply:
Too much work for the PR. The goal is to fix them at one point.